### DISCUSSION TITLE
An issue about creating pull request.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#113406] [Pull Requests]
### Select Topic Area

Question

### Body

I can't create a pull request in the project below, but I've never contributed to this project.
![Screenshot_20240320_165512](https://github.com/community/community/assets/163728725/62a1b764-f247-4e2f-b52b-d81e181d8898)

**Comment by ca-kraa** [comment]
do you have fork before?

**Reply by ghost** [reply]
Yes, Before create PR I ready have fork.

**Reply by ca-kraa** [reply]
check ur repo after fork, maybe in there has option for Pull Request
